recipes/jupyter_resource_usage_proxy/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "jupyter_resource_usage_proxy" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_resource_usage_proxy-{{ version }}.tar.gz
+  sha256: 5d474bd3c78271d6924c4c202da89da01a1a3c176027796513567f950c6e92ab
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+    - jupyter_server >=1.0.0
+    - httpx >=0.23.1
+
+test:
+  imports:
+    - jupyter_resource_usage_proxy
+  commands:
+    - pip check
+  requires:
+    - pip
```

Personally preference. Though please do whatever is easier

```suggestion
test:
  requires:
    - pip
  imports:
    - jupyter_resource_usage_proxy
  commands:
    - pip check
```